requirements/requirements_isolated.txt

```diff
@@ -1,3 +1,3 @@
-psutil==5.2.2
-pexpect==4.2.1
+psutil==5.4.3     # same as AWX requirement
+pexpect==4.4.0    # same as AWX requirement
```

Comment:
@ryanpetrello looks like `pexpect` version seems to have diverged in the main requirements vs. the isolated requirements as-of the last set of dependency updates.